cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -421,7 +428,7 @@ def certificates_list_handler(request, course_key_string):
                 'course_modes': course_modes,
                 'certificate_web_view_url': certificate_web_view_url,
                 'is_active': is_active,
-                'is_global_staff': GlobalStaff().has_user(request.user),
+                'is_global_staff': has_admin_access or GlobalStaff().has_user(request.user),
```

Comment:
nit:
```suggestion
                'is_global_staff': GlobalStaff().has_user(request.user) or has_admin_access,
```

Reply:
the sequence will only matter in-case you create a function and I believe we should.